lib/filters/filter.js

```diff
@@ -44,7 +45,7 @@ function isBerWriter (ber) {
 
 function mixin (target) {
   target.prototype.toBer = function toBer (ber) {
-    if (!ber || !isBerWriter(ber)) { throw new TypeError('ber (BerWriter) required') }
+    if (!isBerWriter(ber)) { throw new TypeError('ber (BerWriter) required') }
```

Comment:
Nit: I prefer explicit checks over coercion negates where possible. I'm sure we have plenty of this shorthand syntax throughout the code, but I'd like to address them as we get to them.

```suggestion
    if (isBerWriter(ber) === false) { throw new TypeError('ber (BerWriter) required') }
```